cupy/indexing/scatter.py

```diff
@@ -0,0 +1,28 @@
+def scatter_update(a, indices, v, axis=0):
+    """Replaces specified elements of an array with given values.
+
+    ``v`` needs to be broadcastable to shape
+        ``a.shape[:axis] + indices.shape + a.shape[axis+1:]``.
+
+    Args:
+        a (ndarray): An array that gets updated.
+        indices (array-like): Indices of elements that this function takes.
```

Comment:
It would be better to explain that each element in `indices` also accepts a negative value.